src/lib.rs

```diff
@@ -105,17 +113,27 @@ pub fn query_osc_buffer<'b, MS: Into<u64> + Copy>(
                     return Err(XQError::NotAnOSCResponse); // EOF
                 }
                 // the sequence must start with a ESC (27) and end either with a ESC or BEL (7)
+                // then, we'll get an 'n' back from the "fence"
                 for i in bytes_written..bytes_written + bytes_read {
                     let b = buffer[i];
                     match osc_start_idx {
                         None => {
-                            if b == 27 {
+                            if b == ESC as u8 {
                                 osc_start_idx = Some(i);
                             }
                         }
                         Some(start_idx) => {
-                            if b == 27 || b == 7 {
-                                return Ok(&buffer[start_idx + 1..=i]);
+                            if b == ESC as u8 || b == /* BEL */ 7 {
```

Comment:
If you define a const for ESC, maybe make it consistent with one for BEL ?

Reply:
Good call